index.html

```diff
@@ -33,6 +33,22 @@
         font-size: 14pt;
       }
     </style>
+<meta property="og:type" content="article" /> 
+<meta property="og:title" content="Nudge Your Friends to Unlike Fossil Fuels" />
+<meta property="og:description" content="https://UnlikeFossilFuels.com 💔🛢🔥🌍⏳
+Unlike the facebook pages of companies that promote fossil fuels and nudge your friends to do the same. #ClimateEmergency" />
+<meta property="og:image" content="LINK TO THE IMAGE FILE" />
```

Comment:
can you provide an image with the pull request? Maybe a screenshot of the emojis?

Reply:
done
image is here : https://i.imgur.com/CmdzNp9.png****